ui/apps/dashboard/src/components/Navigation/OrganizationDropdown.tsx

```diff
@@ -108,7 +108,7 @@ function OrganizationDropdownMenuItem(props: {
       asChild
       className="p-2 font-medium text-slate-400 outline-none hover:bg-transparent focus:text-white"
     >
-      <Link href={props.href as Route}>
+      <Link href={props.href as Route} prefetch={true}>
```

Comment:
When does this prefetch, when the dropdown is open and rendered or when the page loads? Mostly curious of when something gets fetched as I could see us loading a lot if we leaned into this pattern.

Reply:
This prefetches when they enter the viewport, so not until the dropdown is opened. 

I was going to say that the browser will automatically throttle prefetching to make sure it will not interfere with user activity, but I see Next did not use html link + prefetch for this. 😢  In practice I'd say it's fairly self-limiting as you have to turn on a full route prefetch for a link and it's pretty rare that you have more than a half dozen important links from a view the need prefetching.